src/app/components/client/toolbar/Toolbar.module.scss

```diff
@@ -1,10 +1,14 @@
 @use "../../../tokens";
 
 .toolbar {
+  background: tokens.$color-white;
```

Comment:
Blocking issue: I think this is causing unintended changes for the non-redesign toolbar.

![image](https://github.com/user-attachments/assets/20489822-51b2-4ad1-9b5b-bbba5a9f6a2e)

Reply:
Actually, this doesn't quite look in place with the redesigned menu either?

Reply:
Reverted the background change (see https://github.com/mozilla/blurts-server/pull/5671#discussion_r1981288584).

The empty `<Toolbar>` on mobile is not optimal but was a result of [MNTOR-3675](https://mozilla-hub.atlassian.net/browse/MNTOR-3675). This is not a regression introduced by this PR. I’ll run this by UX and expect this to get resolved by the upcoming changes to the settings.